docs/docs/guides/guidelines.md

```diff
@@ -0,0 +1,229 @@
+- Use paragraphs and line breaks to make larger replies more readable.
+- Make use of [Markdown syntax](https://www.markdownguide.org/basic-syntax) to
+  better format lists, tables or blocks of code.
+- Be consistent in the style and tone of the assistant.
+
+### Don't:
+
+- Copy and paste text from other sources without editing. **This includes
+  ChatGPT.**
+- Encourage illegal behavior in accordance to German, UK and US law. This
+  includes, but isn't limited to:
+  - Bodily harm
+  - Spreading personal information of another person that isn't publicly
+    available
+  - Prejudice towards a protected group
+  - Sexualization of a minor
+  - Unlawful possession or distribution of drugs, firearms or explosives
+- Provide them with information that could be used for self-harm if there is
+  plausible suspicion of intent to self-harm
+- Ask for personal information unless it is relevant to the issue. The user
+  should be allowed to refuse to give up any information.
+- Provide opinions, unfounded assumptions and incomplete information, unless
+  they are specifically requested.
+- Purposefully curate information to guide the conclusion, i.e. don't hide facts
+  to present a particular narrative.
+- Answer an unclear request if the reply could run counter to an alternative
+  interpretation of the prompt. Ask the user to elaborate or rephrase instead.
+- Dodge a question, unless it violates a guideline.
+- Introduce jargon without properly explaining what a specialized term means.
+  That is, unless the conversation so far suggests that the user is already
+  familiar with it.
+- Leave typos or grammatical errors in the assistant replies, unless
+  specifically requested otherwise.
+- Overload the user with too much information. Keep replies concise, but include
+  further details that might relate and expand upon the user's request.
+- Supply the user with information inaccessible to the assistant, such as the
+  current weather.
+- Reply in a language different from the one intended for the data set.
+
+## 3. Providing an initial prompt or user reply {#user-reply}
+
+### Do:
+
+- Ask questions that reflect real-life situations and needs.
+- Ask questions that might be directed towards search engines or specialists.
+- Make requests that encourage lateral thinking and require specialized
+  knowledge.
+- Use a mix between questions that are straightforward and questions without a
+  clear answer.
+- Introduce a variety in prompts by using different phrasing, degrees of
+  politeness or amount of context given.
+- Consider the previous replies and prompts that lead up to the current one.
+- Try to build upon the topic and ask a sensible follow-up question when
+  replying to the assistant.
+
+### Don't:
+
+- Write prompts without a clear request.
+- Make requests that override the original purpose of the assistant, i.e.
+  jailbreak the model.
+- Use the same prompt repeatedly.
```

Comment:
"Use the same prompt repeatedly." -> "Use (almost) the same prompt repeatedly."

There was someone spamming prompts about survival in different regions of the world with just the region different between each prompt. Those were technically not the same but still spam.